neusomatic/python/train.py

```diff
@@ -213,7 +210,7 @@ def train_neusomatic(candidates_tsv, validation_candidates_tsv, out_dir, checkpo
         model_dict = net.state_dict()
         # 1. filter out unnecessary keys
         # pretrained_state_dict = {
-        #     k: v for k, v in pretrained_state_dict.items() if k in model_dict}
+        # k: v for k, v in pretrained_state_dict.items() if k in model_dict}
         if "module." in pretrained_state_dict.keys()[0] and "module." not in model_dict.keys()[0]:
```

Comment:
Is there only one key in `pretrained_state_dict`? Is it the case that if `module.` is in a key, it is in all keys? Or is it that the dictionary has keys in sorted order?

Reply:
yes. If "model." is in a key it is in all of them.